src/core_modules/capture-core/components/Tooltips/TooltipOrgUnit/TooltipOrgUnit.component.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from 'react';
+import { Tooltip } from '@dhis2/ui';
+
+type Props = {
+    orgUnitName: string,
+    ancestors?: Array<{| displayName: string, id: number |}>,
+    tooltip?: string,
+};
+
+export const TooltipOrgUnit = ({ orgUnitName, ancestors = [], tooltip }: Props) => {
+    const ancestorNames = ancestors.map(ancestor => ancestor.displayName);
+    ancestorNames.push(orgUnitName);
+    const orgUnitNameFullPath = ancestorNames.join(' / ');
+    const tooltipParts = tooltip ? tooltip.split(orgUnitName) : [orgUnitName];
```

Comment:
Can we simplify this? Do we need the tooltip to span more than the org unit name?